.github/workflows/validate_staging.yml

```diff
@@ -0,0 +1,21 @@
+name: Validate the OpenAPI file and push to staging
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
Can we make this trigger via workflow dispatch too?

With this we can at anytime trigger the Actions from a branch and redeploy. (useful when multiple people have PRs open)
```suggestion
on:
  pull_request:
    branches: [ main ]
  workflow_dispatch:
```

Reply:
That's interesting, I didn't know we could dispatch from a PR, I thought manual is only possible once merged.

Reply:
I tried this, it seems on a branch it doesn't create a separate workflow button. To address the "trigger anytime" issue you mentioned we can just select re-run from the github actions list. It will have the same effect as a dispatch.

Reply:
<img width="1256" alt="Screen Shot 2022-04-15 at 3 19 05 PM" src="https://user-images.githubusercontent.com/14048184/163575580-38ac425d-8ee0-4f2e-b209-3ec63f17cce8.png">